staging-service-client/src/main/java/com/github/cafdataprocessing/services/staging/client/StagingApi.java

```diff
@@ -38,10 +35,34 @@
  *
  */
 public class StagingApi extends com.github.cafdataprocessing.services.staging.client.internal.StagingApi {
+    private final String USER_AGENT_HEADER_NAME = "User-Agent";
+    private final String USER_AGENT_HEADER_VALUE = "caf-staging-service-java-client/1.0.0-SNAPSHOT";
     private final String PUT_API_PATH = "/batches/";
 
-    public StagingBatchResponse createOrReplaceBatch(final String batchId, final Stream<MultiPart> uploadData)
-            throws ApiException, FileNotFoundException, IOException {
+    public StagingApi(final String basePath)
+    {
+        final ApiClient apiClient = getApiClient();
+        apiClient.setBasePath(basePath);
+        apiClient.setUserAgent(USER_AGENT_HEADER_VALUE);
+        /*apiClient.getHttpClient()
+                 .interceptors()
+                 .add(
+                     new Interceptor() {
+                        @Override
+                        public Response intercept(final Chain chain) throws IOException {
+                            final Request request = chain.request();
+                            final Request newRequest = request.newBuilder()
+                                    .header(USER_AGENT_HEADER_NAME, USER_AGENT_HEADER_VALUE)
+                                    .build();
+                            return chain.proceed(newRequest);
+                        }
+                    }
+                );*/
```

Comment:
Why is this code commented out?  Remove it if it is not used.

Reply:
Removed.